src/app/market/shared/orders/orders.component.ts

```diff
@@ -93,6 +102,16 @@ export class OrdersComponent implements OnInit, OnDestroy {
     )
   }
 
+  filter(): void {
+    console.log(this.filters.status);
+    this.loadOrders();
+  }
+
+  clear(): void {
+    this.filters.status = '*';
```

Comment:
This seems pretty fragile, when removing the filters, clear() must always be called.

When doing the call, can't you just
```
this.bid.search(this.profile.address, this.type, this.filters.status || '*' , this.filters.search || '')
```

Default values should be passed like that.

Reply:
Perhaps turn it into a get():

```
get filterStatus() {
  return this.filters.status || '*'
}

Reply:
idem for search

Reply:
I think we have default is already their so we can use that as well ?